WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -1277,9 +1277,7 @@ private extension AztecPostViewController {
             }
         }
 
-        if post.blog.isGutenbergEnabled,
-            let postContent = post.content,
-            postContent.count > 0 && post.containsGutenbergBlocks() {
+        if post.blog.isGutenbergEnabled, post.isContentEmpty() || post.containsGutenbergBlocks() {
```

Comment:
`isContentEmpty` considers a post empty [if it contains an empty paragraph block](https://github.com/wordpress-mobile/WordPress-iOS/blob/edce2beb2cc01df3c66942b2b61139cee3892ccf/WordPress/Classes/Models/BasePost.m#L109-L113), but I think that's OK here.

If the user finds themself with the classic editor open on a post that only contains an empty paragraph block, I can't think of a good reason not to allow them to switch over the the block editor.